scss/components/_richtext.scss

```diff
@@ -28,4 +28,20 @@
       padding-left: 1rem;      
     }
   }
+
+  table {
+    @extend .simple-table;
+  }
+
+  thead {
+    @extend .simple-table__header;
+  }
+
+  tr {
+    @extend .simple-table__row;
```

Comment:
Name of placeholder `.simple-table__row` should be written in all lowercase letters with hyphens instead of underscores